custom-vu/src/main/kotlin/jces1209/vu/ScenarioSimilarities.kt

```diff
@@ -218,33 +218,76 @@ class ScenarioSimilarities(
         browseIssueTypes: Action,
         browseProjectRoles: Action
     ): List<Action> {
-        val exploreData = listOf(browseProjects, browseFilters, browseBoards)
-        val spreadOut = mapOf(
-            createIssue to 0, // 5 if we can mutate data
-            workAnIssue to 55,
-            manageProjects to 5,
-            projectSummary to 5,
-            browseProjects to 5,
-            browseBoards to 5,
-            viewBoard to 30,
-            workOnDashboard to 5,
-            workOnSprint to 0, // 3 if we can mutate data
-            browseProjectIssues to 5,
-            workOnBacklog to 0, // 3 if we can mutate data
-            workOnSearch to 5,
-            workOnTopBar to 5,
-            bulkEdit to 0, // 5 if we can mutate data
-            workOnTransition to 5,
-            workOnWorkflow to 5,
-            browseFieldScreens to 5,
-            browseFieldConfigurations to 5,
-            browseCustomFields to 5,
-            browseIssueTypes to 5,
-            browseProjectRoles to 5
-        )
-            .map { (action, proportion) -> Collections.nCopies(proportion, action) }
-            .flatten()
-            .shuffled(seededRandom.random)
-        return exploreData + spreadOut
+        val readTrafficShapeConfig = System.getenv("readTrafficShapeConfig")
+
+        if (readTrafficShapeConfig.isNullOrEmpty()) {
```

Comment:
Maybe we can use the first mapOf as a default, then override with values from the config file. Or if the Colibiri team agree to it, we put these values in a config file, which could be the default.